interface/globals.php

```diff
@@ -403,10 +403,15 @@ function undoMagicQuotes($array, $topLevel=true) {
 
 // 1 = send email message to given id for Emergency Login user activation,
 // else 0.
-$GLOBALS['Emergency_Login_email'] = $GLOBALS['Emergency_Login_email_id'] ? 1 : 0;
+$GLOBALS['Emergency_Login_email'] = (
+  array_key_exists('Emergency_Login_email_id', $GLOBALS)
+  && $GLOBALS['Emergency_Login_email_id']
+)
+  ? 1
+  : 0;
```

Comment:
Only true if you are 100% sure the return value is NOT going to be **0**, **"0"**, **""**, or **NULL**.